documentation/components.md

```diff
@@ -11,75 +11,137 @@ const Parent = () => (
   </section>
 );
 
-const Label = ({ greeting, children }) => (

```

Comment:
I don't think "good" examples in the docs should ever use object destructuring for props.

Reply:
I agree I honestly don't know how I let that one sneak in. I did start at points doing a lot of copy and paste from React docs.